docs/sources/write/shortcodes/index.md

```diff
@@ -407,6 +407,69 @@ Produces:
 
 {{< docs/public-preview product="public-preview-feature" featureFlag="its-feature-flag" >}}
 
+## Docs/reference
+
+The `docs/reference` shortcode lets you specify different destinations for the same link that depend on where you publish the source file.
+Use this shortcode for links when you reuse content from one repository to more than one documentation set.
+
+You set all possible destinations in one part of the file, usually at end of the file, like a footer.
+
+For example, a page in versioned Grafana documentation is also mounted in the Grafana Cloud documentation.
+The page in Grafana should link to the Grafana dashboards page but the page in Grafana Cloud should link to the Grafana Cloud dashboards page.
+
+Set the reference at the end of the page as follows:
+
+```markdown
+{{%/* docs/reference */%}}
+[dashboards]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/dashboards"
+[dashboards]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/dashboards"
+{{%/* /docs/reference */%}}
+```
+
+The content within the shortcode tags is as follows:
+
+`[LABEL]: "PROJECT PATH PREFIX -> REFERENCE"`
+
+- _`LABEL`_ - The label you'll use in the reference-style links in the file.
```

Comment:
@jdbaldry , why does the text in lines 433-442 appear blue?

Reply:
You mean in GitHub? I don't see blue in the local preview. I guess something is confusing the GitHub syntax highlighter.

Reply:
Yeah, in GH. Ok to ignore then?

Reply:
<img width="665" alt="Screenshot 2024-03-06 at 14 39 19" src="https://github.com/grafana/writers-toolkit/assets/4599384/b1d87e45-689e-45a3-b6db-b6ec7e7a2dfe">

Yeah, it looks fine locally